server/routes/induction/common/wantToAddQualificationsController.ts

```diff
@@ -71,8 +76,18 @@ const assessmentsGroupedByTypeSortedByDateDesc = (assessments: Array<Assessment>
   return assessmentsByType
 }
 
-const createWantToAddQualificationsForm = (): WantToAddQualificationsForm => {
+const createWantToAddQualificationsForm = (inductionDto: InductionDto): WantToAddQualificationsForm => {
```

Comment:
Needed to change this function so that the form object was correctly built, with its `wantToAddQualifications` correctly set. Because its an inferred value that we dont actually store in the Induction:
* the answer is Yes if there are already qualifications on the Induction (ie. you must have previously answered yes in order to add the qualifications)
* the answer is No if there is an empty array of qualifications on the Induction (ie. there is an array of qualifications, but it is empty. This means you have previously been asked about qualifications and you said No)
* the answer is undefined if the induction doesnt have a previousQualifications or qualification object. IE. its a new induction and you've not been asked about qualifications yet.